discord/guild.py

```diff
@@ -3931,6 +3932,38 @@ async def widget(self) -> Widget:
 
         return Widget(state=self._state, data=data)
 
+    WidgetStyle = Literal['shield', 'banner1', 'banner2', 'banner3', 'banner4']
```

Comment:
I wouldn't advise putting this in the namespace as then you can do Guild.WidgetStyle which sits a little weird with me